editions/tw5.com/tiddlers/gettingstarted/GettingStarted.tid

```diff
@@ -15,7 +15,7 @@ Brief instructions for getting started on the different platforms and configurat
 
 Troubleshooting:
 
-* Don't attempt to use the browser ''File''/''Save'' menu option to save changes (it doesn't work)
+* Don't attempt to use the browser ''File''/''Save'' menu option to save changes (it doesn't work).
```

Comment:
Oh I hadn't thought of that. Good catch @pmario though I don't think GitHub lets you redact a review. Any case this PR should probably be closed.